pkg/model/dev_test.go

```diff
@@ -1063,43 +1063,56 @@ func Test_ExpandEnv(t *testing.T) {
 		value         string
 		expandIfEmpty bool
 		result        string
+		expectedErr   error
 	}{
+		{
```

Comment:
adding this test to `ExpandEnv` because the failure case was not covered yet by tests